src/binio.cpp

```diff
@@ -14,7 +14,7 @@ size_t fwrite_big(void *buffer,size_t size,size_t count, FILE *stream)
 
 	st=mx/((unsigned long long) size);
 	n=count/st;
-	for(j=0;j<n;j++) 
+	for(j=0;j<n;j++)
```

Comment:
Please drop these stray white-space changes (here and below).

Reply:
Aren't those white spaces extraneous? My editor automatically strips them (on save). Many editors will do the same. But if you want, I can remove those changes.

Reply:
Please remove these changes.  They are unrelated to the purpose of the commit and so will cause confusion when looking at the log in the long-term.